### PR TITLE
The Medicaling: The Zombining

### PR DESCRIPTION
**Pull request by maaacha**
huge ridiculous PR. takes a ton of zombie code, medical code and other miscellaneous fixes from BLACKSTONE. too many for me to properly document, so i'll just link the last medical rework PR. https://github.com/Blackstone-SS13/BLACKSTONE/pull/1103/

<details>
<summary>general changes</summary>

- lesser miracle now heals wounds sequentially
- fixes spell names showing on mouseover
- refactors zombie code, zombie npcs are now aggressive. probably need to be decapped to be killed.
- refactors all wounds, wound rolls, wound datums significantly
- introduces surgery mechanics
- reworks some spell granting procs
- adds a few spells that are currently not granted
- doctor role + building
- pilgrim doctor class
</details>

<details>
<summary>new wound mechanics</summary>

- eyestab = blinds foe in relevant eye
- tongue stab = tears off tongue, can't speak
- nose stab = disfigurement
- most wounds that have bleed clot, reducing bleed rate over time
- healing spells now heal wounds sequentially
- bandages are more effective
- grabbing halves limb bleed rate. much better than before
- testicular torsion
- neck fractures make you paralyzed
- ear stab = deafness
- skullstab/skullcrack = paralyzed
- mouth stab/mouth fracture = speech gargling
- bones can be dislocated
- makes embeds less awful
</details>

<details>
<summary>new items</summary>

- Super Needle: infinite use needle but cannot be used to repair items
- doctor outfit
- pilgrim doctor outfit
- full surgery gear + bag
- leech and the parasite - extracting and injecting blood
- little knee hammer for people with good medicine to diagnose people with
- cautery iron stops bleeding very fast when heated

</details>

<details>
<summary>surgery guide</summary>

Weak intent is needed to do surgery, to avoid happy little accidents.

Rib fractures:
Scalpel, forceps, speculum, forceps, tend bruises, use foreceps to remove tools, sew incision

Limb attachment:
Scalpel, forceps, speculum, bone saw, hold limb in hand and click on area, use forceps to remove tool, sew wound shut and use cautery to rapidly seal wounds caused by limb attachment

Limb fractures:
Scalpel, speculum, bonesaw/bone forceps, use forceps to remove tools, stitch and bandage incision then have them rest.

there's other surgeries such as organ manipulation etc but i cbfed writing up how to do those you can experiment/code dive : )
</details>

### CODE REVIEW

File: code/modules/jobs/job_types/roguetown/nobility/lady.dm [hunk 1]
```diff
@@ -20,11 +20,27 @@
...
+
 /datum/outfit/job/roguetown/lady/pre_equip(mob/living/carbon/human/H)
 	. = ..()
-	ADD_TRAIT(H, RTRAIT_SEEPRICES, TRAIT_GENERIC)
-	beltl = /obj/item/keyring/queen
-//	head = /obj/item/clothing/head/roguetown/hennin
+	ADD_TRAIT(H, TRAIT_SEEPRICES, TRAIT_GENERIC)
+	ADD_TRAIT(H, TRAIT_NOBLE, TRAIT_GENERIC)
+	ADD_TRAIT(H, TRAIT_NUTCRACKER, TRAIT_GENERIC)
+	beltl = /obj/item/roguekey/manor
```
Comment (BadAtThisGame302) [inline code/modules/jobs/job_types/roguetown/nobility/lady.dm:43]: Why would you remove the Queens keyring? She has no access to her own chambers now.

File: code/modules/jobs/job_types/roguetown/nobility/magician.dm [hunk 2]
```diff
@@ -25,16 +27,15 @@
...
 	shoes = /obj/item/clothing/shoes/roguetown/shortboots
 	backr = /obj/item/storage/backpack/rogue/satchel
 	armor = /obj/item/clothing/suit/roguetown/shirt/robe/black
 	cloak = /obj/item/clothing/cloak/black_cloak
 	id = /obj/item/clothing/ring/gold
 	belt = /obj/item/storage/belt/rogue/leather/plaquesilver
 	beltr = /obj/item/keyring/mage
-	neck = /obj/item/clothing/neck/roguetown/talkstone
-	pants = /obj/item/clothing/under/roguetown/tights/random
+	id = /obj/item/clothing/ring/gold
```
Comment (BadAtThisGame302) [inline code/modules/jobs/job_types/roguetown/nobility/magician.dm:38]: Two golden rings.

Comment (BadAtThisGame302) [reply code/modules/jobs/job_types/roguetown/nobility/magician.dm:38]: Also why remove the Magicians talkstone?

**Comment by Plaguewalker** [comment]
More of suggestion lessen the bloat on roles, maybe roll the feldspar into an acolyte of pestra instead while also removing their nobility trait? As, well as increasing the physicker role slots for the time being?

**Comment by Kabramen** [comment]
An suggestion to the feldspar clinic: it needs more decorations both outside and in the lobby, to indicate that the place is, well, the clinic. 

While playing as feldspar in the first test round, i've notice many people were confused at the place since it looked more like a normal shop, and at the same time others didn't even knew about its location, which made that most people went to the church to get healed instead.

A simples sign similar to this one could help players to identify the clinic more easily. maybe it could have a caduceus as symbol, or even Pestra's if she has any.
image

**Comment by BadAtThisGame302** [comment]
You can't target the hair anymore?

**Pull request merged**